app/src/main/java/com/cornellappdev/transit/ui/screens/RouteScreen.kt

```diff
@@ -215,6 +264,22 @@ fun RouteScreen(
             .background(color = DividerGrey)
             .fillMaxSize(), content = {})
     }
+
+    if (openBottomSheet.value) {
+        ModalBottomSheet(
+            sheetState = sheetState,
+            onDismissRequest = {
+                closeModal()
+            },
+            dragHandle = { },
+            content = {
+                RouteOptionsSearchSheet(routeViewModel, onCancelClicked = {
+                    closeModal()
+                })
+            }
+        )
+    }
+
 }
```

Comment:
Hmm, I'm a bit sus of this code to make the sheet no longer render. Does this just make the sheet instantly disappear instead of slide down?

You might be able to use ModalBottomSheetLayout instead if so. I think a sheet disappearing might look a bit confusing. Or maybe there's some other field you can use to check if the sheet should be visible or not as in not pushed down all the way?